experanto/datasets.py

```diff
@@ -438,8 +253,13 @@ def initialize_statistics(self) -> None:
                     self._experiment.devices[device_name].root_folder / "meta/stds.npy"
                 )
                 if device_name == "responses":
-                    idx = stds[0, :] < 1  # response std shape: (1, n_neurons)
-                    stds[0, idx] = 1  # setting stds which are smaller than 1 to 1
+                    # same as in neuralpredictors before
+                    # https://github.com/sinzlab/neuralpredictors/blob/2b420058b2c0c029842ba739829114ddfa0f8b50/neuralpredictors/data/transforms.py#L375-L378
+                    threshold = 0.01 * np.nanmean(stds)
+                    idx = stds[0, :] < threshold  # response std shape: (1, n_neurons)
+                    stds[0, idx] = (
+                        threshold  # setting stds which are smaller than 1 to threshold
+                    )
```

Comment:
Why the parenthesis?
And the comment should be the following, no?
```suggestion
                    stds[0, idx] = threshold  # setting stds which are smaller than threshold to threshold
```

Reply:
parenthesis are coming from black due to the very long comments, so they will be back. But the comment should be updated indeed, thanks!